pkg/config/system_probe.go

```diff
@@ -256,6 +256,12 @@ func InitSystemProbeConfig(cfg Config) {
 	cfg.BindEnvAndSetDefault(join(smNS, "process_service_inference", "enabled"), false, "DD_SYSTEM_PROBE_PROCESS_SERVICE_INFERENCE_ENABLED")
 	cfg.BindEnvAndSetDefault(join(smNS, "process_service_inference", "use_windows_service_name"), true, "DD_SYSTEM_PROBE_PROCESS_SERVICE_INFERENCE_USE_WINDOWS_SERVICE_NAME")
 
+	cfg.BindEnvAndSetDefault(join(spNS, "http_map_cleaner_interval_in_s"), 300)
```

Comment:
Won't this always cause a deprecation message to be printed, because `IsSet` doesn't ignore default values?

Reply:
indeed.

tried to put the default value in the new config
```go
	cfg.BindEnv(join(spNS, "http_map_cleaner_interval_in_s"))
	cfg.BindEnvAndSetDefault(join(smNS, "http_map_cleaner_interval_in_s"), 300)
```

but it breaks the backward compatibility
can we detect is the set value is "default"?

Reply:
Not with the current config interface. You can use the `applyDefault` function to help with this though.

Reply:
Excellent catch.
Made the fixes